src/routes/Root/scripts.js

```diff
@@ -0,0 +1,65 @@
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
Please refactor all of this to belong in the relevant React components, using React conventions.